### PR TITLE
tested & fixed contact-us & newsletter

### PR DESCRIPTION
**Pull request by avinashvidyarthi**
- Tested newsletter & contact-us flow
- Added mailer credentials through .env file
- Added googleAccountKey.json file for blob uploads into gs bucket
- updated .gitignore & .env.example file

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,9 +9,12 @@ DB_URL=
...
 EMAIL_PORT=
+EMAIL_SECURE=true #boolean
 EMAIL_USER=
 EMAIL_PASSWORD=
 
 # Jwt
 JWT_SECRETE=
 JWT_EXPIRE_IN=
+
+GOOGLE_APPLICATION_CREDENTIALS="./googleAccountKeys.json" #path to key file
```
Comment (InSantoshMahto) [inline .env.example:20]: remove this

GOOGLE_APPLICATION_CREDENTIALS is not required in production or staging environment.
use this env only on local machine

**Review by InSantoshMahto — CHANGES_REQUESTED** [review]
do the required changes

**Comment by InSantoshMahto** [comment]
added single space just after comment operators like # and //

example 

# your comments goes here
// js comments here ..

File: .env.example (hunk 1 above)
Comment (avinashvidyarthi) [reply .env.example]: Removed GOOGLE_APPLICATION_CREDENTIALS from .env.example file

**Comment by avinashvidyarthi** [comment]
Removed GOOGLE_APPLICATION_CREDENTIALS from .env.example file

**Review by InSantoshMahto — APPROVED** [review]

**Pull request merged**